src/qt/forms/receiverequestdialog.ui

```diff
@@ -50,7 +50,7 @@
      <property name="minimumSize">
       <size>
        <width>0</width>
-       <height>50</height>
+       <height>176</height>
```

Comment:
We should really try to avoid relying on hardcoded heights. The necessary hight varies based on things such as font size, DPI, styling.

Reply:
Indeed. Can we make the whole dialog floating (dynamic height)?